questions/magic-potion.md

```diff
@@ -0,0 +1,63 @@
+# Magic Potion
+
+## Question
+
+Welcome to **Magic Potion**! Here is the problem statement:
+
+Sabrina the witch has a list of magical ingredients that she uses to create potions. Each ingredient has a unique magical power level.
+
+The list of ingredients is sorted in increasing order based on their magical power levels.
+
+One day, Sabrina received an order for a magic potion. The order specified the exact magical power level required. Now, Sabrina needs to choose some ingredients from her list such that the sum of their magical power levels equals the required level.
```

Comment:
the idea is really fun but the explanation of how to use magic power levels and why this aspect is relevant is confusing. i think making the objective would be helpful